app/src/main/res/navigation/nav_graph.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/nav_graph"
+    app:startDestination="@id/listFragment">
+
+    <fragment
+        android:id="@+id/mapFragment"
+        android:name="com.nearbynest.app.ui.map.MapFragment"
+        android:label="fragment_map"
+        tools:layout="@layout/fragment_map" />
+    <fragment
+        android:id="@+id/listFragment"
```

Comment:
Id should be self descriptive. List is very generic word.

Reply:
Yes , List is bit more generic.
Used `propertyListFragment` instead `listFragment`.